dali/test/python/jax/test_integration.py

```diff
@@ -123,3 +123,26 @@ def test_dali_sequential_tensors_to_jax_array():
                     shape[1:],  # TODO(awolant): Explain/fix shape consistency
                     batch_id * batch_size + i,
                     np.int32))
+
+
+def test_dali_sequential_iterator_to_jax_array():
```

Comment:
You may also consider extending test_fw_iterators.py.

Reply:
Done, I added some tests to test_fw_iterators. Tests there are mostly about various parameters of the iterators.

I am leaving tests specific to JAX and device support in JAX specific files.